includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -198,6 +198,7 @@ public function get_settings(): WP_REST_Response {
 				'payment_request_button_type'       => $this->wcpay_gateway->get_option( 'payment_request_button_type' ),
 				'payment_request_button_theme'      => $this->wcpay_gateway->get_option( 'payment_request_button_theme' ),
 				'is_saved_cards_enabled'            => $this->wcpay_gateway->is_saved_cards_enabled(),
+				'is_card_present_eligible'          => $this->wcpay_gateway->is_card_present_eligible(),
```

Comment:
Looks like we have mix of tabs/spaces in the array formatting, let's stick to the same style as in the first items of the array.